src/bridgestanR.cpp

```diff
@@ -23,23 +23,24 @@ void bs_model_info_R(bs_model** model, char const** info_out) {
 }
 void bs_param_names_R(bs_model** model, int* include_tp, int* include_gq,
                       char const** names_out) {
-  *names_out = bs_param_names(*model, *include_tp, *include_gq);
+  *names_out = bs_param_names(*model, (*include_tp != 0), (*include_gq != 0));
```

Comment:
Why not change the function parameter from `int* include_tp` to `bool* include_tp`?

`bridgestan/R/R/bridgestan.R` (that's funny) is passing arguments as `as.logical(include_tp), as.logical(include_gq)` so aren't we still oddly juggling ints and bools?

Reply:
I checked this again this morning, and R's documentation states that `int*` is correct:

```
     The mapping of the types of R arguments to C or Fortran arguments
     is

       R          C                Fortran          
       integer    int *            integer          
       numeric    double *         double precision 
       - or -     float *          real             
       complex    Rcomplex *       double complex   
       logical    int *            integer   
```

The reason is that logical values in R can not just be `TRUE` and `FALSE`, but also `NA`:

```
     Logical values are sent as ‘0’ (‘FALSE’), ‘1’ (‘TRUE’) or ‘INT_MIN
     = -2147483648’ (‘NA’, but only if ‘NAOK = TRUE’), and the compiled
     code should return one of these three values: however non-zero
     values other than ‘INT_MIN’ are mapped to ‘TRUE’.
```

Reply:
I did not know this.  Thanks.